src/main/resources/messages_en_US.properties

```diff
@@ -1,58 +1 @@
-typeMismatch.jpaExampleEntity.shoeSize=must be a valid number
-typeMismatch.java.util.Date=must be a valid date
-typeMismatch.java.lang.Integer=must be a valid Integer
-typeMismatch.java.lang.Long=must be a valid Long
-typeMismatch.number=must be a valid number
-typeMismatch=You have entered incorrect data on this page.  Please fix (Catches all not found)
-
-#Candidate form trigger validations
-formTrigger.candidate.example=Example candidate validation
-formTrigger.candidate.example.error=Error running example candidate validation. Please try again.
-
-
-#Client Corporation form trigger validations
-formTrigger.clientcorporation.example=Example client corporation validation
-formTrigger.clientcorporation.example.error=Error running Example Candidate Validation. Please try again.
-
-#Client Contact form trigger validations
-formTrigger.clientcontact.example=Example client contact validation
-formTrigger.clientcontact.example.error=Error running Example client contact validation. Please try again.
-
-#Placement form trigger validations
-formTrigger.placement.example=Example placement validation
-formTrigger.placement.example.error=Error running Example placement validation. Please try again.
-formTrigger.placement.roe=The following rules of engagement are missing: {0}. These items need to be completed in order for the placement to be approved.
-formTrigger.placement.roe.error=Error running Placement ROE validation. Please contact your system administrator.
-formTrigger.placement.example2=Status cannot be updated to Approved
-formTrigger.placement.example2.error=Error running Example 2 placement validation. Please try again.
-
-
-#Placement Change Request form trigger validations
-formTrigger.placementchangerequest.example=Example placement Change Request validation
-formTrigger.placementchangerequest.example.error=Error running Example placement Change Request validation. Please try again.
-
-
-#Job form trigger validations
-formTrigger.job.example=Example job validation
-formTrigger.job.example.error=Error when running Example job validation. Please try again.
-
-#Job Submission form trigger validations
-formTrigger.jobsubmission.example=Example job Submission validation
-formTrigger.jobsubmission.example.error=Error when running Example job Submission validation. Please try again.
-
-
-#Note form trigger validations
-formTrigger.note.example=Example note validation
-formTrigger.note.example.error=Error when running Example note validation. Please try again.
-formTrigger.note.example.error=Error when running Example note validation. Please try again.
-
-
-gui.combo.initialValue=NOT_SELECTED
-gui.combo.initialDisplay=--- Select ---
-
-validation.field.isrequired={0} is required
-validation.field.numeric=Please enter valid numeric value for {0}
-
-#test property for unit test making sure this .properties file is being read to messageSource
-core.test.property=core property
+# Add any validation messages you want in English
```

Comment:
Did this not fail unit tests???
I thought we used `core.test.property` in one of them. But maybe that was the SDK

Reply:
Ah. I see you moved it to test resources. I like that a lot more